var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -16,8 +16,9 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
     maintainers = ["ryandanehy", "CameronRutherford", "pelesh"]
 
     version(
-        "1.4.1", commit="ea607c685444b5f345bfdc9a59c345f0f30adde2", submodules=True, preferred=True
+        "1.5.0", commit="227f49573a28bdd234be5500b3733be78a958f15", submodules=True, preferred=True
     )
```

Comment:
Since the latest version will default to `preferred=True` perhaps this can be simplified [and remove `preferred=True` from here]